core/App/components/buttons/Button.tsx

```diff
@@ -19,7 +19,7 @@ export interface ButtonProps {
 }
 
 export const Button: React.FC<ButtonProps & React.RefAttributes<HTMLInputElement | undefined>> = forwardRef(
-  ({ title, buttonType, accessibilityLabel, testID, onPress, disabled = false }, ref: any) => {
+  ({ title, buttonType, accessibilityLabel, children, testID, onPress, disabled = false }, ref: any) => {
```

Comment:
I think you can just use the `PropsWithChildren` type then you don't have to pass children as an explicit prop

Reply:
Are there any examples in the codebase? I tried changing the type to `PropsWithChildren<ButtonProps & React.RefAttributes<HTMLInputElement | undefined>>` and `PropsWithChildren<ButtonProps> & React.RefAttributes<HTMLInputElement | undefined>` but I'm running into issues with react complaining that `Objects are not valid as a React child`